website/notifications/emails.py

```diff
@@ -78,7 +80,17 @@ def email_digest(subscribed_user_ids, uid, event, **context):
 }
 
 
-def notify(uid, event, **context):
+def notify(uid, event, user, node, timestamp, **context):
+    """
+    :param uid: node's id
+    :param event: type of notification
+    :param user: user triggering notification
+    :param node: the node
+    :param timestamp: time
+    :param context: optional variables specific to templates
+        target_user: used with comment_replies
```

Comment:
Could we just explicitly pass `comment_replies` as the event type instead of computing it based on `context['target_user']`. The `**context` is meant to be used for the mako templates, and removing comment-specific code from this code was the original purpose of this PR.